build/configs/rtl8730e/rtl8730e_download.sh

```diff
@@ -100,6 +100,8 @@ function board_download()
 	else
 		if [[ $TTYDEV == *"USB"* ]]; then
 			#echo "UART download"
+			echo "$3" >> setting.txt
+			echo "$2" >> setting.txt
```

Comment:
What's the `$3` and `$2`? Can we use some variables instead of using argument directly?

Reply:
this one is for checking, i remove it